packages/pwa-kit-dev/src/utils/script-utils.ts

```diff
@@ -300,8 +387,13 @@ export const createBundle = async ({
                         existsSync(path.join(extendsTemplate, item))
                     )
                 }
+
                 bundle_metadata = {
-                    dependencies: {...dependencies, ...devDependencies},
+                    dependencies: getDependencies(
+                        dependencies,
+                        devDependencies,
+                        getProjectDependencyTree()
+                    ),
```

Comment:
```suggestion
                    dependencies: {
                        ...dependencies,
                        ...devDependencies,
                        ...getPwaKitDependencies(getProjectDependencyTree())
                    },
```
nit: `dependencies` and `devDependencies` aren't used in `getDependencies` except to merge the result, so they don't really need to be passed to the method.